src/v2_0/rws_client.cpp

```diff
@@ -304,6 +304,20 @@ std::string RWSClient::generateFilePath(const FileResource& resource)
   return Services::FILESERVICE + "/" + resource.directory + "/" + resource.filename;
 }
 
+// Try finding xml tag with class="code" attribute containing retcode. Does not throw.
+std::optional<int> tryParseRetcode(std::string const& content) noexcept
+{
+  std::optional<int> result;
+  try
+  {
+    auto xml = parseXml(content);
+    std::string code_str = xmlFindTextContent(xml, XMLAttribute{"class", "code"});
+    if (!code_str.empty())
+        result = std::stoi(code_str);
+  } catch (...) {}
```

Comment:
I am very much in favor of
```suggestion
  } 
  catch (Poco::XML::XMLException const&) 
  {
  }
```